faucet/valve.py

```diff
@@ -164,6 +165,17 @@ def _dump_lag_status_event_sock(self):
                             'port_no': port.number,
                             'status': port.dyn_lacp_up}})
 
+    def _dump_learned_hosts_event_sock(self):
+        learned_macs = []
```

Comment:
I think better to keep this as a map by mac address rather than a list -- same as the underlying data structure.  Same information, just ordered slightly differently.

Reply:
Nevermind, if the goal here is to preserve the content structure of the L2_LEARN event, then this (a list) is likely the best way to do it.

Reply:
Safe to resolve?